contrib/inventory/proxmox.py

```diff
@@ -219,6 +221,7 @@ def main():
     parser.add_option('--username', default=os.environ.get('PROXMOX_USERNAME'), dest='username')
     parser.add_option('--password', default=os.environ.get('PROXMOX_PASSWORD'), dest='password')
     parser.add_option('--pretty', action="store_true", default=False, dest='pretty')
+    parser.add_option('--trust-invalid-certificates', action="store_true", default=False, dest='validate')
```

Comment:
command line options are not usable from `ansible-playbook`, you probably also want a environment variable as for username and password.